medusa/indexers/indexer_base.py

```diff
@@ -214,6 +214,26 @@ def _save_images(self, series_id, images):
                 res: resolution such as `1024x768`, `original`, etc
                 id: the image id
         """
+        def penalize_images(images):
+            """Penalize images that have a rating count of less then 5 and images that are not withing the highest
+            category of resolution, if there are multiple."""
+            images = list(images)  # We don't need to the generator.
+            banner_types = set([image['bannertype2'] for image in images])
+
+            # Try to sort the banner types based on quality (might not be needed)
+            banner_types = sorted(banner_types)
+
+            for image in images:
+                if image['ratingcount'] is not None and image['ratingcount'] < 5:
+                    image['rating'] -= 1
```

Comment:
I don't like this type of rating.  Personally i'd go with a weighted average.

Reply:
I understand that's up for debate.